extensions/interactions/Continue/directives/continue-validation.service.ts

```diff
@@ -50,6 +50,13 @@ export class ContinueValidationService {
         message: 'The button text should not be empty.'
       });
     }
+
+    if (customizationArgs.buttonText.value.unicode.length > 20) {
+      warningsList.push({
+        type: AppConstants.WARNING_TYPES.CRITICAL,
+        message: 'The button text should be less than 20 characters.'
+      });
+    }
```

Comment:
Do you think it's better to do the validation here rather than in `Continue.py` (like the other validations in this PR)?

Reply:
@EricZLou
Yeah, I thought the same and tried to put the validation in the `Continue.py` file itself but it didn't seem to work that way. Also, I thought that there is no `Error message` to display and it would be more efficient to put the validation in here.